gocd/templates/peanutbutter.jsonnet

```diff
@@ -0,0 +1,35 @@
+local peanutbutter = import './pipelines/peanutbutter.libsonnet';
+local pipedream = import 'github.com/getsentry/gocd-jsonnet/libs/pipedream.libsonnet';
+
+local pipedream_config = {
+  // Name of your service
+  name: 'peanutbutter',
+  // Peanutbutter should only run on `us` and `de`.
+  exclude_regions: ['s4s', 'customer-1', 'customer-2', 'customer-3', 'customer-4', 'customer-5', 'customer-6', 'customer-7'],
```

Comment:
Hopefully there is a better way to achieve this.

Reply:
We might want to deploy this to s4s, even though it is not *useful* there:
https://github.com/getsentry/ops/pull/10699#pullrequestreview-2077092688

Reply:
Fair point, I'll adjust it.